src/main/java/reposense/parser/CsvParser.java

```diff
@@ -22,14 +25,11 @@
     protected static final String COLUMN_VALUES_SEPARATOR = ";";
     protected static final Logger logger = LogsManager.getLogger(CsvParser.class);
 
-    private static final String ELEMENT_SEPARATOR = ",";
     private static final String OVERRIDE_KEYWORD = "override:";
     private static final String MESSAGE_UNABLE_TO_READ_CSV_FILE = "Unable to read the supplied CSV file.";
-    private static final String MESSAGE_MALFORMED_LINE_FORMAT = "Warning! line %d in CSV file, %s, is malformed.\n"
-            + "Content: %s";
+    private static final String MESSAGE_MALFORMED_LINE_FORMAT = "Warning! line %d in CSV file, %s, is malformed.";
```

Comment:
Is this the downside of using the api? We can't provide the content that is malformed?

Reply:
Not that it's not possible, but printing the content will also print lots of irrelevant details. Printing it results in this : -

`CSVRecord [comment=null, mapping={Repository's Location=0, Branch=1, File formats=2, Ignore Glob List=3, Ignore standalone config=4, Ignore Commits List=5}, recordNumber=1, values=[https://github.com/reposense/testrepo-Alpha.git, master, , , , 2fb6b9b2dd9fa40bf0f9815da2cb0ae8731436c7;c5a6dc774e22099cd9ddeb0faff1e75f9cf4f151;cd7f610e0becbdf331d5231887d8010a689f87c7;768015345e70f06add2a8b7d1f901dc07bf70582]]`

Reply:
Hmm... it would be best to if we can, because line number alone can cause some discrepancy in understanding 

E.g. some may think the line number begins from the first line of entry (excluding the header)

Reply:
Made the changes